src/trumbowyg.js

```diff
@@ -1486,6 +1486,37 @@ jQuery.trumbowyg = {
         getRangeText: function () {
             return this.range + '';
         },
+        expandRange: function () {
+            var t = this,
+                documentSelection = t.doc.getSelection();
+
+            if (documentSelection.rangeCount) {
+                var ranges = [];
+                for (var i=0; i<documentSelection.rangeCount; i+=1) {
+                    var range = documentSelection.getRangeAt(i);
+                    var ancestor = range.commonAncestorContainer;
+
+                    if (ancestor.nodeType === 1) {
+                        if (range.startContainer.parentNode !== ancestor && documentSelection.containsNode(range.startContainer.parentNode, true)) {
```

Comment:
Use a variable to store `range.startContainer.parentNode` because like that, uglifier can't minify this code.

So the minified from this code should be something like this:

```
if (r.startContainer.parentNode !== ancestor && d.containsNode(r.startContainer.parentNode, true)) {
```

But if you use a variable to store the parentNode it should be minified to something like that:

```
var p = r.startContainer.parentNode;
if (p !== ancestor && d.containsNode(p, true)) {
```

And if you see other optimizations like that you can apply them.

The gulpfile log the trumbowyg.min.js size and this size should not go over 21kB